src/pages/library/common.js

```diff
@@ -192,8 +195,32 @@ export const selectionActionComponent = (selectedData, setSelectedData) => {
           }, 'Cancel'),
           h(ButtonPrimary, {
             style: { textTransform: 'none', fontSize: 14 },
-            onClick: () => {}
+            onClick: () => {
+              setShowExportModal(true)
+            }
           }, ['Save to a workspace'])
+        ]),
+        showExportModal &&
+        h(Modal, {
+          title: 'Save to a Terra workspace',
+          onDismiss: () => setShowExportModal(false),
+          okButton: () => {
+            Nav.history.push({
+              pathname: Nav.getPath('import-data'),
+              search: `?url=https://jade.datarepo-dev.broadinstitute.org/&snapshotId=idid&snapshotName=${datasetName}&format=snapshot`
```

Comment:
Just to make it more obvious, and using the correct data repo url:
```suggestion
              search: `?url=${getConfig().dataRepoUrlRoot}&snapshotId=REPLACE_ME&snapshotName=${datasetName}&format=snapshot`
```